source/developers-guide/shopware-config/index.md

```diff
@@ -38,6 +38,10 @@ need to specify options in your `config.php` if you want to override the default
 But keep in mind that most of these options should only be used for __debugging and testing__ 
 and should be removed for your live system.
 
+### Environment-specific config
+
+To be able to set different configs for different environments, you can place a file called `config_ENVIRONMENT.php` in the Shopware root directory. `ENVIRONMENT` should be replaced with the environment the kernel gets initialized and defaults to `production`, e.g. `config_production.php`. The environment-specific config file is preferred over the normal one.
```

Comment:
this is already described in this blog article: https://developers.shopware.com/blog/2016/01/26/configuring-multiple-shopware-environments/#config-loading

maybe we could just link this here

Reply:
My opinion is that we should have a central point of documentation for the config.php.
But we can additionally link the blog post here, as this is a more advanced.

Reply:
that sounds good :+1:

Reply:
I added the link to the blog post and squashed my commits.
This should now be good to go!